sdk/test/akkaserverless-test.js

```diff
@@ -15,6 +15,7 @@
  */
 
 const should = require('chai').should();
+const expect = require('chai').expect;
```

Comment:
should we converge to one testing style?
I am not strongly opinionated but I do prefer to stitch with one "per-cdebase"

Reply:
Sure, no strong preference, just didn't find API to express the same assertion with `should`, do you know how that would look for the this array should have the same elements as that array?

Reply:
Should be able to just rewrite in `should` format:

```js
comp.getEntity().getForwardHeadersList().should.have.same.members([
  'x-my-header'
]);
```